docs/Documentation/Systems/Swift/running.md

```diff
@@ -489,7 +480,7 @@ You can get an interactive session on a compute node with the salloc command, as
 ```bash
 [nrmc2l@swift-login-1 ~]$ salloc  --account=hpcapps   --exclusive    --time=01:00:00   --ntasks=16           --nodes=1 --partition=debug
 ```
-
+(this salloc returns error: slurmstepd: error: couldn't chdir to '/home/xhe': No shuch file or directory: going to /tmp instead)
```

Comment:
This command works fine for me. Can you test again? Might be an issue with your home directory on Swift